### DISCUSSION TITLE
Mobile view removes images from features

### DISCUSSION THREAD

**Discussion by snaqvi1990** [greenelab/lab-website-template#235] [Q&A] [answered]
Thanks for the great template! I am modifying the template's features on the research page, and they look great on web but when viewed on mobile, the images for each feature are gone (screenshot attached). Is there a way to keep the image in mobile and have it be on top of the text for each feature?

Thanks,
Sahin
![Image 1-30-24 at 3 15 PM](https://github.com/greenelab/lab-website-template/assets/7546459/5da03d38-fd89-4c70-81a5-53ad1611aa41)

![image](https://github.com/greenelab/lab-website-template/assets/7546459/ccefa6ed-234b-417c-9b7a-fc0cf6480e91)

**Comment by vincerubinetti** [comment, marked as answer]
Unfortunately this looks like a bug, specifically with mobile Safari (I'm assuming that's what you're using? Safari is usually the problematic browser, Chrome and Firefox usually do things right).

A quick fix:

Under `@media (max-width: 800px) .feature-image {` in `/_styles/feature.scss`, change `width: unset;` to `width: 100%;`. That should fix it. 

I'll capture this bug in an issue.

**Comment by snaqvi1990** [comment]
It actually looked the same on both Safari and Chrome, but your fix worked great! Thanks

**Reply by vincerubinetti** [reply]
FYI if you're on ios, Chrome still runs Safari under the hood, so that would be why you're still seeing the issue.

**Reply by snaqvi1990** [reply]
Got it, thanks for the quick reply!
